cmd/edgectl/aes_install.go

```diff
@@ -168,7 +289,15 @@ func (i *Installer) Perform(kcontext string) error {
 	i.show.Printf("-> Installing the Ambassador Edge Stack %s\n", aesVersion)
 
 	// Attempt to talk to the specified cluster
-	// TODO: Figure out cluster info to be passed along
+	// TODO: Figure out cluster info to be passed along.
```

Comment:
Most likely outdated.

Reply:
Same here. GitHub has this marked as outdated.
![image](https://user-images.githubusercontent.com/1765854/74874400-e7284b00-532e-11ea-9b48-17783c1df1ca.png)